index.html

```diff
@@ -0,0 +1,152 @@
+                </ul>
+            </nav>
+        </header>
+        <div class="headline">
+            <div class="textdiv"><h1>I’m Alexandra <br>
+                Glad to see you!</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through 
+                  some of my work and experience! If you like what you see and have a project you need coded,
+                   don’t hestiate to contact me.</p>
+                   <h3 class="pcol">LET’S CONNECT</h3>
+                    <ul class="iconlist">
+                        <li><img class="icons" src="twitter.svg" alt="twitter"></li>
+                        <li><img class="icons" src="linkedinicon.svg" alt="linkedin"></li>
+                        <li><img class="icons" src="m.svg" alt="mimage"></li>
+                        <li><img class="icons" src="github.svg" alt="github"></li>
+                        <li><img class="icons" src="victory.svg" alt="victory"></li>
+                    </ul>
+                </div>
+            </div>
+        <div class="container">
+            <div class="imgdiv">
+                <img class="projects" src="image1.png" alt="Nature">
+            </div>
+            <div class="tonic">
+                    <h1 class="popheading">Tonic</h1>
+                    <p class="popp13">CANOPY <img class="dot" src="dot.png" alt="dot"> 
+                        <span class="popp13 col">Back End Dev</span> <img class="dot" src="dot.png" alt="dot"> 
+                         <span class="popp13 col">2015</span></p>
+                    <br>
+                    <p class="popp15">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <span class="htcj">HTML</span> 
+                    <span class="htcj">CSS</span>
+                    <span class="htcj">Javascript</span><br><br>
+                      <div class="btn">
+                        <a href="image1.png" target="_blank">See Project</a>
+                      </div>
+            </div>    
+           </div>
+           <div class="container2">
+            <div class="imgdiv2">
+                <img class="projects" src="image2.png" alt="Nature">
+            </div>
+            <div class="multi">
+                    <h1 class="popheading">Multi Post Series</h1>
+                    <p class="popp13">CANOPY <img class="dot" src="dot.png" alt="dot"> 
+                        <span class="popp13 col">Back End Dev</span> <img class="dot" src="dot.png" alt="dot"> 
+                         <span class="popp13 col">2015</span></p>
+                    <br>
+                    <p class="popp15">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <br>
+                    <span class="htcj2">HTML</span> 
+                    <span class="htcj2">CSS</span>
+                    <span class="htcj2">Javascript</span><br><br>
```

Comment:
- [ ] [REQUIRED] Please implement the [previous reviewer's](https://github.com/baqar-abbas/Portfolio-Project-Final-version/pull/1/files#r1073049408) suggestion 🙏🏼 The technology in each category(e.g., "JavasScript", "HTML" and "CSS") should be a list `ul`, and each of them a `li`, not a `span` 😃